resources/js/beatmapsets-show/info.tsx

```diff
@@ -198,7 +198,7 @@ export default class Info extends React.Component<Props> {
                   <React.Fragment key={`${tag.name}-${i}`}>
```

Comment:
unrelated but the `${i}` isn't needed for this one

or maybe can even use tag.id instead